### DISCUSSION TITLE
backtesting.py

### DISCUSSION THREAD

**Discussion by dbogatic** [kernc/backtesting.py#408] [General]
Hello, 
I have a question, just to make sure, regarding backtesting.py data when fitting the model. Is it taking into account overfitting and look ahead bias? Thank you for the hard work and for making this available to the public! 

Dragan

**Comment by kernc** [comment]
What do you mean by fitting the model? Backtesting.py itself doesn't fit, nor prevent overfitting to, any data. As long as none of your "indicators" (wrapped in [`Strategy.I`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I)) have lookahead bias, you're probably safe.

**Reply by dbogatic** [reply]
Thank you for responding.
